### PR TITLE
catalog_hover

### PR DESCRIPTION
**Pull request by ISRLv2021**
- [DEMO LINK](https://ISRLv2021.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -16,6 +16,12 @@
     margin-bottom: 48px;
   }
 
+  &__card:hover {
+    transform: scale(120%);
+    color: #00acdc;
+    transition: 300ms;
```
Comment (dellachaise) [inline src/styles/_catalog.scss:22]: `transition: 300ms` better place in &__card selector

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -16,6 +16,12 @@
     margin-bottom: 48px;
   }
 
+  &__card:hover {
+    transform: scale(120%);
+    color: #00acdc;
```
Comment (dellachaise) [inline src/styles/_catalog.scss:21]: please change the color to the one asked in the task(it should be darker)

File: src/styles/_header.scss [hunk 3]
```diff
@@ -38,4 +38,10 @@
     font-size: 12px;
     font-weight: 500;
   }
+
+  &__link:hover {
+    color: #00acdc;
+    transition: 0.5s;
```
Comment (dellachaise) [inline src/styles/_header.scss:44]: `transition duration` better write in &__link

**Review by dellachaise — CHANGES_REQUESTED** [review]

File: src/styles/_catalog.scss (hunk 1 above)
Comment (ISRLv2021) [reply src/styles/_catalog.scss:22]: catalog__card--hover do ypu mean like this one?

File: src/styles/_catalog.scss (hunk 2 above)
Comment (ISRLv2021) [reply src/styles/_catalog.scss:21]: yep, it shoud be #34568b

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**